### ISSUE TITLE
Accessibility testing

### ISSUE THREAD

**Issue by amandavisconti** [scholarslab/scholarslab.org#217]
Run through testing and add info to /accessibility page (#9)
https://github.com/btholt/postcss-colorblind

**Comment by jeremyboggs** [comment]
This fails pretty spectacularly, using Chrome's built-in accessibility audits. #519 is just one example of that. There are a number of places where we lack `alt` attributes entirely.

**Comment by amandavisconti** [comment]
@jeremyboggs I added #526 to work on adding alt attributes. Are there other major things, in addition to alt's and the search JSON impact on page load time? (Are these things we should hold off releasing until we fix, or release the site and then I'll prioritize addressing them?)
